internal/controller/pgupgrade/jobs_test.go

```diff
@@ -264,3 +265,55 @@ spec:
 status: {}
 	`))
 }
+
+func saveEnv(t testing.TB, key string) {
```

Comment:
:pick: Took me a minute to figure out why this function is needed. A comment might help here.